diff --git a/cmake/Modules/Packages/KOKKOS.cmake b/cmake/Modules/Packages/KOKKOS.cmake
index 7f23a6f777..3c1a600578 100644
--- a/cmake/Modules/Packages/KOKKOS.cmake
+++ b/cmake/Modules/Packages/KOKKOS.cmake
@@ -81,7 +81,7 @@ else()
   # build KOKKOS internal libraries as static libraries but with PIC, if needed
   if(BUILD_SHARED_LIBS)
     set(BUILD_SHARED_LIBS_WAS_ON YES)
-    set(BUILD_SHARED_LIBS OFF)
+    # set(BUILD_SHARED_LIBS OFF)
   endif()
   if(CMAKE_REQUEST_PIC)
     set(CMAKE_POSITION_INDEPENDENT_CODE ON)
